src/sgame/components/SpikerComponent.cpp

```diff
@@ -2,13 +2,33 @@
 
 static Log::Logger logger("sgame.spiker");
 
+/** Delay between shots. */
+constexpr int   COOLDOWN = 5000;
+/** Shoot prematurely whenever this much damage is expected. */
+constexpr float DAMAGE_THRESHOLD = 60.0f;
+/** Total number of missiles to shoot. Some might be skipped if they endanger allies.
+ *  Actual values is +/- SPIKER_MISSILEROWS. */
+constexpr float MISSILES = 26;
+/** Number of rows from which to launch spikes. */
+constexpr int   MISSILEROWS = 4;
+/** 0.0: Spikes are shot upwards, 1.0: Spikes are shot sideways. */
```

Comment:
I believe the comment on `ROWOFFSET` is not accurate.

Reply:
What's wrong with it? Is it the other way round?

Reply:
I'll actually just remove it, because only `0.5f` makes sense with the rest of the code.